.cilibs/test_floodgate_against_running_spinnaker_instance.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+FLOODGATE_EXTRA_PARAMS=$1
+
+echo Test Floodgate against running Spinnaker instance
+/floodgate/bin/floodgate --version
```

Comment:
Maybe you can add some info about what is happening before executing this commands?

Reply:
Added printing as was in .circleci/config.yml